lib/src/tar/import.rs

```diff
@@ -434,7 +433,7 @@ impl<'a> Importer<'a> {
         let mut contents = vec![0u8; n as usize];
         entry.read_exact(contents.as_mut_slice())?;
         let contents: glib::Bytes = contents.as_slice().into();
-        let contents = variant_new_from_bytes(OSTREE_XATTRS_FORMAT, contents, false);
+        let contents = Variant::from_bytes::<&[(&[u8], &[u8])]>(&contents);
```

Comment:
Note how we were previously passing a string `a(ayay)` and now encode the Variant type via Rust generics.